content/docs/optimizing-performance.md

```diff
@@ -1,57 +1,57 @@
 
-<img src="../images/docs/devtools-prod.png" style="max-width:100%" alt="React DevTools on a website with production version of React">
+<img src="../images/docs/devtools-prod.png" style="max-width:100%" alt="React DevTools dalam situs dengan versi produksi React">
 
-If you visit a site with React in development mode, the icon will have a red background:
+Jika Anda mengunjungi situs yang menggunakan React dalam mode pengembangan, ikon ekstensi akan memiliki latar belakang merah:
 
-<img src="../images/docs/devtools-dev.png" style="max-width:100%" alt="React DevTools on a website with development version of React">
+<img src="../images/docs/devtools-dev.png" style="max-width:100%" alt="React DevTools dalam situs dengan versi pengembangan React.">
 
-It is expected that you use the development mode when working on your app, and the production mode when deploying your app to the users.
+Anda diharapkan menggunakan mode pengembangan ketika mengerjakan aplikasi Anda, dan mode produksi ketika men-*deploy* aplikasi Anda kepada pengguna.
 
-You can find instructions for building your app for production below.
+Anda dapat menemukan instruksi untuk membangun aplikasi Anda untuk mode produksi di bawah.
 
 ### Create React App {#create-react-app}
 
-If your project is built with [Create React App](https://github.com/facebookincubator/create-react-app), run:
+Jika proyek Anda dibangun menggunakan [Create React App](https://github.com/facebookincubator/create-react-app), jalankan:
 
 ```
 npm run build
 ```
 
-This will create a production build of your app in the `build/` folder of your project.
+Ini akan membuat versi produksi untuk aplikasi Anda di dalam folder `build/` di proyek Anda.
 
-Remember that this is only necessary before deploying to production. For normal development, use `npm start`.
+Ingat bahwa ini hanya diperlukan sebelum men-*deploy* ke produksi. Untuk pengembangan normal, gunakan `npm start`.
 
-### Single-File Builds {#single-file-builds}
+### *Single-File Build* {#single-file-builds}
 
-We offer production-ready versions of React and React DOM as single files:
+Kami menyediakan versi siap produksi dari React dan React DOM sebagai *file* tunggal:
 
 ```html
 <script src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
 <script src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
 ```
 
-Remember that only React files ending with `.production.min.js` are suitable for production.
+Ingat bahwa hanya *file* React yang berakhir dengan `.production.min.js` yang layak digunakan untuk produksi.
 
 ### Brunch {#brunch}
 
-For the most efficient Brunch production build, install the [`terser-brunch`](https://github.com/brunch/terser-brunch) plugin:
+Untuk *build* produksi yang efisien menggunakan Brunch, instal *plugin* [`terser-brunch`](https://github.com/brunch/terser-brunch):
 
 ```
 # If you use npm
```

Comment:
Shouldn't we translate this? It's only a comment anyway

Reply:
Good catch!